useful-scripts/misc-test-utils/src/index.ts

```diff
@@ -28,24 +39,134 @@ const start = async () => {
+          message:
+            "would you like the uploadSubject function to exit as soon as a 'QUEUED' status is received? (this can be useful for uploading lots of subjects quickly)",
+        });
+
+        for (let i = 0; i < numOfSubjects.number; i++) {
+          await uploadSingleSubject(release.id, fast.shouldBeFast);
+        }
+        break;
+
+      case 'create new methodology':
+        const publication = await inquirer.prompt({
+          name: 'id',
+          type: 'input',
+          message: 'publication ID from existing publication',
+          prefix: '>',
+        });
+        await createMethodology(publication.id);
+        break;
+
+      case 'generate lorem ipsum content text block (methodology & release)':
+        const methodologyOrRelease = await inquirer.prompt({
+          name: 'type',
+          type: 'checkbox',
+          choices: ['Methodology', 'Release'],
+        });
+
+        const times = await inquirer.prompt({
+          name: 'number',
+          type: 'number',
+          message: 'How many content blocks do you want to create?',
+          prefix: '>',
+          default: 1,
+        });
+
+        switch (methodologyOrRelease.type[0]) {
+          case 'Methodology':
+            // create a methodology content block
+            const publicationExists = await inquirer.prompt({
+              name: 'exists',
+              type: 'confirm',
+              message: 'Do you have an existing publication?',
+              prefix: '>',
+            });
+
+            if (publicationExists.exists) {
+              const methodology = await inquirer.prompt({
+                name: 'id',
+                type: 'input',
+                message: 'methodology ID from existing publication',
+                prefix: '>',
+              });
+
+              for (let i = 0; i < times.number; i++) {
+                await addMethodlogyTextContentBlock(methodology.id);
+              }
+            }
+            console.log(
+              'Creating new publication, release & methodology with lorem ipsum content',
+            );
+            const { publicationId } = await createPublicationAndRelease();
+            const methodologyId = await createMethodology(publicationId);
+            for (let i = 0; i < times.number; i++) {
+              await addMethodlogyTextContentBlock(methodologyId);
+            }
+            break;
+
+          case 'Release':
+            const releaseExists = await inquirer.prompt({
+              name: 'exists',
+              type: 'confirm',
+              message: 'Do you have an existing release?',
+              prefix: '>',
+            });
+
+            if (!releaseExists.exists) {
+              const { releaseId } = await createPublicationAndRelease();
+
+              for (let i = 0; i < times.number; i++) {
+                await addReleaseTextContentBlock(releaseId);
+              }
+            } else {
+              const rl = await inquirer.prompt({
```

Comment:
No biggie, but why `rl`?

Reply:
This was just to avoid a shadow of the `release` variable. I've changed this to `existingRelease` to be a bit more descriptive